java-strings/src/test/java/com/baeldung/string/MatchWordsUnitTest.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.string;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MatchWordsUnitTest {
+
+    private final String[] words = {"hello", "Baeldung"};
+    private final String inputString = "hello there, Baeldung";
+    private final String wholeInput = "helloBaeldung";
+
+    @Test
+    public void givenText_whenCallingStringContains_shouldMatchWords() {
+        final boolean result = MatchWords.containsWords(inputString, words);
+        assertThat(result).isEqualTo(true);
```

Comment:
```suggestion
        assertThat(result).isTrue();
```

Reply:
I will make these changes in another PR (no perms to push changes)